src/controller/ReadInteraction.h

```diff
@@ -218,11 +224,13 @@ CHIP_ERROR ReportEvent(Messaging::ExchangeManager * apExchangeMgr, EndpointId en
 template <typename DecodableEventType>
 CHIP_ERROR ReadEvent(Messaging::ExchangeManager * exchangeMgr, const SessionHandle & sessionHandle, EndpointId endpointId,
                      typename TypedReadEventCallback<DecodableEventType>::OnSuccessCallbackType onSuccessCb,
-                     typename TypedReadEventCallback<DecodableEventType>::OnErrorCallbackType onErrorCb)
+                     typename TypedReadEventCallback<DecodableEventType>::OnErrorCallbackType onErrorCb,
+                     bool fabricFiltered = false)
```

Comment:
What would this even mean?  There is no concept of fabric-filtered read for events, last I checked.

Reply:
```
• Fabric scoped events SHALL be encapsulated in one of two forms:
◦ A fabric-idx data type field at the top-level of the event definition
◦ A struct field within the event containing a fabric-idx data type field
```

Seems we have one?

Reply:
We have a concept of fabric scoping for events, but not fabric filtering on read.  The only thing that has a concept of fabric filtering on reads is list-typed attributes.

Reply:
Fabric-filtering as a concept does only apply to attributes. However, fabric-scoped events I presume are not readable from fabrics other than the one that generated that event. So presumably, event reads are always 'fabric filtered' by definition?

Reply:
Yes, which is why we shouldn't have a client API for controlling the fabric-filtering for events: passing `false` here does not affect the server behavior compared to `true`.

Reply:
Thanks for clarification, removed this flag from event read.